src/api/AccessRight/types.ts

```diff
@@ -23,7 +23,8 @@ export enum Action {
   RAW_READ_QUERY = 'rawReadQuery', // Create a read query
   RAW_WRITE_QUERY = 'rawWriteQuery', // Create a read/write query
   RUN_CUSTOM_ACTION = 'runCustomAction', // Execute a custom action
-  WRITE_CUSTOM_ACTION = 'writeCustomAction' // Edit, update and delete a custom action
+  WRITE_CUSTOM_ACTION = 'writeCustomAction', // Edit, update and delete a custom action
+  MANAGE_CUSTOM_ACTION = 'manageCustomAction' // Create, edit, update and delete a custom action
```

Comment:
It's worth documenting more accurately the difference between these two rights here. Otherwise, this looks confusing someone without context.

From my understanding:
* `WRITE_CUSTOM_ACTION`: create a custom action + update/delete owned custom actions
* `MANAGE_CUSTOM_ACTION`: `WRITE_CUSTOM_ACTION` + update/delete all non-private custom actions